### DISCUSSION TITLE
README里的思维导图是用什么软件做的呀？

### DISCUSSION THREAD

**Discussion by LHTXQ** [ldqk/Masuit.Tools#82] [Q&A] [answered]
如题。

**Comment by ldqk** [comment, marked as answer]
xmind zen

**Comment by LHTXQ** [comment]
谢啦！回复太快了！

发件人: ***@***.***>
发送时间: 2023年2月20日 9:15
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [ldqk/Masuit.Tools] README里的思维导图是用什么软件做的呀？ (Discussion #82)


xmind zen

―
